### ISSUE TITLE
Update guide for version 18 - unclear (possibly incorrect?) instruction for opting out of caching HTTP requests that require authentication

### ISSUE THREAD

**Issue by Nellas** [angular/angular#57790]
### Describe the problem that you experienced

The update guide for version 18 on angular.dev has the following text as one of the bullet points - 
```
Use includeRequestsWithAuthHeaders: true in withHttpTransferCache to opt-out of caching for HTTP requests that require authorization.
```
Is that statement incorrect?  My understanding is that setting `includeRequestsWithAuthHeaders: true` actually opts-in to caching for HTTP requests that require authorization.
The default behavior (`includeRequestsWithAuthHeaders: false`) is to exclude such requests from being cached.

### Enter the URL of the topic with the problem

https://angular.dev/update-guide?v=17.0-18.0&l=3

### Describe what you were looking for in the documentation

Looking for any unknowns or gotchas while preparing to update from version 17 to 18.

### Describe the actions that led you to experience the problem

Reading through the recommended actions.

### Describe what you want to experience that would fix the problem

If my understanding is correct, then a simple copy change is all :) 
If not - forgiveness! :P

**Comment by JeanMeche** [comment]
I agree, it should say "opt-in".

**Comment by Suryaprakash1024** [comment]
I will provide the fix

**Comment by Suryaprakash1024** [comment]
#57799

**Comment by pkozlowski-opensource** [comment]
Fixed in #57799
